wcc-contentful/lib/wcc/contentful/store/query.rb

```diff
@@ -50,7 +61,16 @@ def initialize(store, content_type:, conditions: nil, options: nil, **extra)
     # @expected The expected value to compare the field's value against.
     # @context A context object optionally containing `context[:locale]`
     def apply_operator(operator, field, expected, context = nil)
+      operator ||= expected.is_a?(Array) ? :in : :eq
       raise ArgumentError, "Operator #{operator} not supported" unless respond_to?(operator)
+      raise ArgumentError, 'value cannot be nil (try using exists: false)' if expected.nil?
+
+      case operator
+      when :in, :nin, :all
+        expected = Array(expected)
```

Comment:
So this one feels like it's the more controversial inference -  
if someone writes the following:
```rb
Page.find_by(something: { in: '1,2,3' })
```
then currently it turns into a `#<Condition @operator=:in @expected=['1,2,3']`, when it really should be `#<Condition @operator=:in @expected=['1', '2', '3']`

But again I'm hesitant to infer that split based on a  test like`expected.include?(',')`

Reply:
yep